<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>pyXCMS</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../:/new/prefix1/icons/games.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>xcms</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QPushButton" name="getFolderBtn" >
                <property name="text" >
                 <string>Select Folder</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_25" >
                  <property name="text" >
                   <string>Data Folder:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="curFolderLE" >
                  <property name="readOnly" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_21" >
                <property name="text" >
                 <string>Files in Directory:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="dirListWidget" >
                <property name="selectionMode" >
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>Grouping Method:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="xcmsMethodCB" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select XCMS Processing Method&lt;/p>&lt;/body>&lt;/html></string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>Parameters:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CustomTable" name="paramTableWidget" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Adjust XCMS Parameters&lt;/p>&lt;/body>&lt;/html></string>
                </property>
                <property name="rowCount" >
                 <number>0</number>
                </property>
                <property name="columnCount" >
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="runXCMSBtn" >
              <property name="toolTip" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Make it happen!&lt;/p>&lt;/body>&lt;/html></string>
              </property>
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset>../../../../:/new/prefix1/icons/applications.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>R output:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="RoutputTE" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>EIC</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="text" >
                 <string>EIC Index:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="eicIndexSlider" >
                <property name="maximum" >
                 <number>150</number>
                </property>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="eicIndexSB" >
                <property name="maximum" >
                 <number>150</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="MPL_Widget" native="1" name="plotWidget" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_7" >
                  <property name="text" >
                   <string>Current Folder:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="eicCurFolderLE" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="readOnly" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_5" >
                  <property name="layoutDirection" >
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text" >
                   <string>m/z Start:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="mzStartSB" >
                  <property name="toolTip" >
                   <string>Upper bound of the EIC extraction range</string>
                  </property>
                  <property name="decimals" >
                   <number>4</number>
                  </property>
                  <property name="maximum" >
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="singleStep" >
                   <double>0.050000000000000</double>
                  </property>
                  <property name="value" >
                   <double>215.150000000000006</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_4" >
                  <property name="text" >
                   <string>m/z Stop:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="mzStopSB" >
                  <property name="toolTip" >
                   <string>Low bound of the EIC extraction range</string>
                  </property>
                  <property name="decimals" >
                   <number>4</number>
                  </property>
                  <property name="maximum" >
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="singleStep" >
                   <double>0.050000000000000</double>
                  </property>
                  <property name="value" >
                   <double>227.150000000000006</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_11" >
                  <property name="text" >
                   <string>RT Start:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rtWidthSB" >
                  <property name="toolTip" >
                   <string>Retention time window of the EIC</string>
                  </property>
                  <property name="decimals" >
                   <number>1</number>
                  </property>
                  <property name="minimum" >
                   <double>1.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="singleStep" >
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>200.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_12" >
                  <property name="text" >
                   <string>RT Stop:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rtWidthSB_Stop" >
                  <property name="toolTip" >
                   <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Set to -1 for the entire spectrum&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
                  </property>
                  <property name="decimals" >
                   <number>1</number>
                  </property>
                  <property name="minimum" >
                   <double>-1.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>100000.000000000000000</double>
                  </property>
                  <property name="singleStep" >
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value" >
                   <double>-1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="rtTypeCB" >
                <property name="toolTip" >
                 <string>Do you want the corrected EIC or the raw values?</string>
                </property>
                <property name="text" >
                 <string>Retrieve Corrected EIC?</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="plotLegendCB" >
                <property name="text" >
                 <string>Plot Legend?</string>
                </property>
                <property name="checked" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="getEICBtn" >
                <property name="toolTip" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Do It!&lt;/p>&lt;/body>&lt;/html></string>
                </property>
                <property name="text" >
                 <string>Fetch EIC</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4" >
       <attribute name="title" >
        <string>Batch Processing</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="loadRPY2BatchBtn" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Load rpy2 Batch Script</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="batchScriptTE" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="runBatchBtn" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Do It!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>Options</string>
       </attribute>
       <widget class="QPushButton" name="killRBtn" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text" >
         <string>Kill R Process</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_HDF5" />
    <addaction name="actionLoad_HDF5" />
   </widget>
   <widget class="QMenu" name="menuF_unctions" >
    <property name="title" >
     <string>F&amp;unctions</string>
    </property>
    <addaction name="actionSave_Results_Table" />
    <addaction name="separator" />
    <addaction name="actionRun_XCMS" />
    <addaction name="separator" />
    <addaction name="actionTest_XCMS" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuF_unctions" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Results_Table" >
   <property name="text" >
    <string>Save Results Table</string>
   </property>
  </action>
  <action name="actionTest_XCMS" >
   <property name="text" >
    <string>Test XCMS</string>
   </property>
   <property name="toolTip" >
    <string>Needs faakho package</string>
   </property>
   <property name="whatsThis" >
    <string>Needs faakho package</string>
   </property>
  </action>
  <action name="actionSave_HDF5" >
   <property name="text" >
    <string>Save HDF5</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_HDF5" >
   <property name="text" >
    <string>Load HDF5</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRun_XCMS" >
   <property name="text" >
    <string>Run XCMS</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTable</class>
   <extends>QTableWidget</extends>
   <header>customTable</header>
  </customwidget>
  <customwidget>
   <class>MPL_Widget</class>
   <extends>QWidget</extends>
   <header>mpl_pyqt4_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>eicIndexSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eicIndexSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>182</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eicIndexSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eicIndexSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
